src/helm/benchmark/scenarios/scenario.py

```diff
@@ -55,7 +56,11 @@ class Input:
     The input of an `Instance`.
     """
 
-    text: str
+    text: str = ""
+    """The text of the input (e.g, passage to summarize, text for sentiment analysis, etc.)"""
+
+    multimodal_content: Optional[MultimediaObject] = None
```

Comment:
Should make terms consistent - object versus content, multimedia versus multimodal...  how about just `multimedia_object` or at least `multimedia_content`?

Reply:
I changed it to `multimedia_content` and updated the PR description with the new names. I left `multimodal` for objects on the Adapter level: `MultimodalPrompt`, `multimodal_prompt` in `Request`, `InContextLearningMultimodalAdapter`, etc. (which seems to make more sense), or do you think it should all be `multimedia`?